Sample.java

```diff
@@ -1,7 +1,78 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
+// Time Complexity : Push - O(1), Pop -> O(1), Min -> O(1)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : not sure about the space complexity as this is what we have to return so will this stack space be included in the space complexity.
```

Comment:
You can ask this question to the interviewer. Generally, if you are not returning the data structure then it is taking extra space. So yes it should be O(n) here.